test/python/tools/test_country_info.py

```diff
@@ -59,3 +59,39 @@ def test_create_country_names(temp_db_with_extensions, temp_db_conn, temp_db_cur
     else:
         assert result_set == {'us' : set(('us', 'us1', 'us2', 'United States')),
                               'fr' : set(('fr', 'Fra', 'Fren'))}
+
+@pytest.mark.parametrize("yaml_file_content", (
+"""
+de:
+    partition: 3
+    names: 
+        name: 
+            default: Deutschland
+""",
+"""
+de:
+    partition: 3
+    languages: de
+    names:
+""",
+"""
+de:
+    partition: 3
+    languages: de
+"""
+))
+def test_load(project_env, def_config, yaml_file_content):
+    (project_env.project_dir / 'country_settings.yaml').write_text(yaml_file_content)
+    
+    country_info._COUNTRY_INFO._info = def_config.load_sub_configuration(
+        (project_env.project_dir / 'country_settings.yaml'))
+    
+    for prop in country_info._COUNTRY_INFO._info.values():
+        if 'languages' not in prop:
```

Comment:
You make assumptions here that the import has worked right. That is not allowed. I.e. you cannot use the result of the function under test to decide which asserts to run. It is better to write 3 separate tests here.